sds/src/scanner/mod.rs

```diff
@@ -832,6 +856,95 @@ mod test {
         assert_eq!(matches.len(), 1);
     }
 
+    #[test]
```

Comment:
It's possible that the path is missing in case a Event is a String. you can probably add a test for this case.

Reply:
I added a test case corresponding to when the feature is disabled: [Split tests and move path sanitization](https://github.com/DataDog/dd-sensitive-data-scanner/pull/53/commits/4c3068390ed5fce0bcadc0f6471fa1586bda910e)